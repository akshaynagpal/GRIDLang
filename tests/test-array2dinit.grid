int gameloop(){
    int[4][4] y;
    print("array2d init success");
    return 0;
}

int checkGameEnd()
{
  return 1;
}
