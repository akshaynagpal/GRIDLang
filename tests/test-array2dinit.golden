array2d init success
